inst/include/sampler.h

```diff
@@ -112,6 +115,8 @@ namespace smc {
         double GetParticleLogWeightN(long n) const { return pPopulation.GetLogWeightN(n); }
         ///Return the unnormalized weight of particle n
         double GetParticleWeightN(long n) const { return pPopulation.GetWeightN(n); }
+        ///Return the unnormalized weight of particle n
```

Comment:
Ah yes you're right, thanks. I meant to say "Return the unnormalised weights of the particles". I'll fix that up now.